src/js/views/patients/schedule/schedule.scss

```diff
@@ -7,6 +7,112 @@
   margin-right: 16px;
 }
 
+.schedule__filter-date {
+  margin-left: auto;
+}
+
 .schedule-list__header {
```

Comment:
I think we should go ahead and make here down in a `schedule-list.scss` file.  They're highly related, but I think the block name being the filename is proving useful.